apache/application.wsgi.mako

```diff
@@ -0,0 +1,21 @@
+import site
+import sys
+import re
+
+site.addsitedir("${python_path}")
```

Comment:
I'm relatively confident with it, yes it should be remove from `apache/wsgi.conf`, I think that better that we have everywhere the same configuration :-)